horizon/templates/app-settings.configmap.yaml

```diff
@@ -56,7 +64,12 @@ data:
   GF_SERVER_ROOT_URL: "/"
   {{- end }}
   GF_DATABASE_TYPE: "postgres"
+  {{- if .Values.multiTenant }}
   GF_DATABASE_NAME: {{ printf "%s_grafana" .Release.Name | quote }}
+  {{- else }}
+  GF_DATABASE_NAME: "grafana"
+  {{- else }}
```

Comment:
We might want to remove the extra `else`

Reply:
Yup. :)